### PR TITLE
Zoho.ZohoMail.Desktop version 1.3.2

### PR DESCRIPTION
**Pull request by kdpuvvadi**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/29502)

### CODE REVIEW

File: manifests/z/Zoho/ZohoMail/Desktop/1.3.2/Zoho.ZohoMail.Desktop.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Zoho.ZohoMail.Desktop
+PackageVersion: 1.3.2
+PackageLocale: en-US
+Publisher: Zoho Corporation
+PackageName: Zoho Mail Desktop
```
Comment (ItzLevvie) [inline manifests/z/Zoho/ZohoMail/Desktop/1.3.2/Zoho.ZohoMail.Desktop.locale.en-US.yaml:8]:
```suggestion
PackageName: Zoho Mail - Desktop
```

File: manifests/z/Zoho/ZohoMail/Desktop/1.3.2/Zoho.ZohoMail.Desktop.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Zoho.ZohoMail.Desktop
+PackageVersion: 1.3.2
+PackageLocale: en-US
+Publisher: Zoho Corporation
```
Comment (ItzLevvie) [inline manifests/z/Zoho/ZohoMail/Desktop/1.3.2/Zoho.ZohoMail.Desktop.locale.en-US.yaml:7]:
```suggestion
Publisher: Zoho Mail
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/z/Zoho/ZohoMail/Desktop/1.3.2/Zoho.ZohoMail.Desktop.locale.en-US.yaml (hunk 2 above)
Comment (kdpuvvadi) [reply manifests/z/Zoho/ZohoMail/Desktop/1.3.2/Zoho.ZohoMail.Desktop.locale.en-US.yaml:7]: i think publisher should be Zoho corporation as in they are the publisher of said app

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
>i think publisher should be Zoho corporation as in they are the publisher of said app

Window Package Manager requires the `Publisher` to match Control Panel.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**